README.md

```diff
@@ -359,6 +359,14 @@ breakpoints: ?Object = {lg: 1200, md: 996, sm: 768, xs: 480, xxs: 0},
 // # of cols. This is a breakpoint -> cols map, e.g. {lg: 12, md: 10, ...}
 cols: ?Object = {lg: 12, md: 10, sm: 6, xs: 4, xxs: 2},
 
+// # of margin. This is a breakpoint -> margin map, e.g. {lg: [10, 10], md: [10, 10], ...}
+margin: [number, number] | ?Object = [10, 10] | {lg: [10, 10], md: [10, 10], sm: [10, 10], xs: [10, 10], xxs: [10, 10]},
```

Comment:
here we should describe default value, it can't be an array and an object in the same time. In the code it's an object for now:

```typescript
margin: {
      lg: [10, 10],
      md: [10, 10],
      sm: [10, 10],
      xs: [10, 10],
      xxs: [10, 10]
    },
```

But I think it'll be better to use an array here (e. g. [10, 10]), because it gives the same result.